### DISCUSSION TITLE
How to collect data for the new light device

### DISCUSSION THREAD

**Discussion by sw0277** [T-Troll/alienfx-tools#328] [General]
When I get some free time, I'd like to follow your instructions on "How to collect data for the new light device" so I can get that info over to you. I read through you're instructions a couple times and noticed that you need to use AWCC  to get the info you need.

Do you have to use AWCC or is there another way? Currently I'm just running Alienfx, AWCC is long gone. If not, I'll reinstall AWCC to get the info needed, I was just trying to avoid putting that worthless software back on my laptop.

**Comment by T-Troll** [comment]
Well... AWCC is only needed for new device class, in case my tools doesn't recognize chip or can't control it.
Adapting to the new model doesn't require it in common, just some manual effort in GUI to map lights (described in beginner's guide).
